packages/block-directory/src/components/downloadable-block-icon/index.js

```diff
@@ -1,25 +1,14 @@
 /**
  * WordPress dependencies
  */
-import { __, sprintf } from '@wordpress/i18n';
 import { BlockIcon } from '@wordpress/block-editor';
 
-function DownloadableBlockIcon( { icon, title } ) {
-	return (
-		<div className="block-directory-downloadable-block-icon">
-			{ icon.match( /\.(jpeg|jpg|gif|png|svg)(?:\?.*)?$/ ) !== null ? (
-				<img
-					src={ icon }
-					alt={ sprintf(
-						// translators: %s: Name of the plugin e.g: "Akismet".
-						__( '%s block icon' ),
-						title
-					) }
-				/>
-			) : (
-				<BlockIcon icon={ icon } showColors />
-			) }
-		</div>
+function DownloadableBlockIcon( { icon } ) {
+	const className = 'block-directory-downloadable-block-icon';
+	return icon.match( /\.(jpeg|jpg|gif|png|svg)(?:\?.*)?$/ ) !== null ? (
+		<img className={ className } src={ icon } alt="" />
```

Comment:
FYI: This was added as a result of a piece of feedback here: https://github.com/WordPress/gutenberg/pull/16524/files#r324222681

Reply:
The alt text, you mean? I think that feedback was about the icon labels. In this case, it sounds repetitive when using a screen reader to have "Icon of [Block Plugin]" right next to the block name, IMO the icon is functionally decorative.

Reply:
Sorry wrong link: https://github.com/WordPress/gutenberg/pull/16524/files#r324268820.

But year, sounds ok.